Solutions/Network Session Essentials/Analytic Rules/PossibleBeaconingActivity.yaml

```diff
@@ -77,21 +77,60 @@ query: |
   let TimeDeltaThreshold = 10;
   let TotalEventsThreshold = 15;
   let PercentBeaconThreshold = 80;
+  let LocalNetworks=dynamic(["169.254.0.0/16","127.0.0.0/8"]);
   _Im_NetworkSession(starttime=ago(querystarttime), endtime=ago(queryendtime))
   | where not(ipv4_is_private(DstIpAddr))
-  | project TimeGenerated,   SrcIpAddr, SrcPortNumber, DstIpAddr, DstPortNumber, DstBytes, SrcBytes
-  | sort by SrcIpAddr asc,TimeGenerated asc, DstIpAddr asc, DstPortNumber asc
+  | where not (ipv4_is_in_any_range(DstIpAddr, LocalNetworks))
```

Comment:
@azurekid can we use - not (ipv4_is_private(DstIpAddr)) for taking only internet traffic ?
https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/ipv4-is-privatefunction

Reply:
@azurekid I just noticed that filter is being used already at line 82. Your change looks good, please ignore my previous comment

Reply:
No worries, according to the documentation some ranges are nor excluded via the private-id function.

Therefore the let statement is used. 👍